script/Arcueid/Day 2/02_00_ARC02_6.txt

```diff
@@ -106,12 +106,12 @@ It's more than just disturbing my sleep.
 [sha:02a7374e2e99695c03133dd1c688eea8ea914281]{
 -- Page 343, Offset 3511.
 -- 犬の遠吠えは屋敷の外から聞こえてくる。
-The dog's howl reaches all the way to my ears from outside the mansion.
+I can hear the dog's howl from all the way outside the mansion.
```

Comment:
I think the awkward phrasing before had been to avoid the possibility of reading this like _Shiki_ is outside the mansion. Old ver def not great though

Reply:
from -> coming from makes it clearer i think